book/src/chap04.asciidoc

```diff
@@ -473,19 +471,19 @@ Uma declaração que lê um arquivo módulo e cria um objeto módulo.
 (((declaração using)))
```

Comment:
O significa em inglês de módulo é "A file that contains a collection of related functions and other definitions."

Eu acho q em português fica: "Um arquivo que contém uma coleção de funções  e outras definições relacionadas."